practice_2/ViewController.swift

```diff
@@ -1,19 +1,128 @@
-//
-//  ViewController.swift
-//  practice_2
-//
-//  Created by user199993 on 6/27/21.
-//
-
 import UIKit
 
+enum ScreenSpace: Int {
+    case top = 0
+    case middle = 1
+    case bottom = 2
+}
+
 class ViewController: UIViewController {
+    var models = [Model]()
+    @IBOutlet weak var tableView: UITableView!
```

Comment:
IBOutlets should be private to avoid leaking UIKit to higher layers ([Private Outlet](https://realm.github.io/SwiftLint/private_outlet.html))

Reply:
Thanks for that point. Done